### PR TITLE
Add support for please in mid sentence

### PR DESCRIPTION
**Pull request by taybenlor**
I haven’t tested this extensively.

Also ignores “can I” and similar

### CODE REVIEW

**Comment by taybenlor** [comment]
Sorry I’m a silly and originally PR’d this to Greg’s repo

File: coffeespecs.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 _OUT_ORDER = ['size', 'iced', 'milk', 'strength', 'decaf', 'type', 'sugar']
 
-ALLOW_TOKENS = ['with', 'please', 'i would like', 'would like', 'can i have', 'may i have', 'thanks', 'a', 'me', 'order', 'mine', 'for', 'the', 'like', 'want']
+ALLOW_TOKENS = ['with', 'please', 'i would like', 'would like', 'can i have', 'may i have', 'thanks', 'a', 'me', 'order', 'mine', 'for', 'the', 'like', 'want', 'can i', 'may i']
```
Comment (katharosada) [inline coffeespecs.py]: This part isn't necessary anymore, since this list is gone. :)

File: test_coffeespecs.py [hunk 2]
```diff
@@ -173,5 +173,8 @@ def testPrint(self):
     c = Coffee('Soy decaf latte with 2 sugars')
     self.assertEquals('Regular Soy Decaf Latte with 2 Sugars', str(c))
 
+    c = Coffee('Can I')
+    self.assertFalse(c.validate())
```
Comment (katharosada) [inline test_coffeespecs.py]: Yeah, sorry, tests got refactored too. 😬

**Comment by spauka** [comment]
No longer an issue :)

**Pull request closed without merging**